src/native/libs/System.Globalization.Native/local-build/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.6)
+
+project(System.Globalization.Native.Local.Build C)
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux") # Only Linux for now
+
+configure_file(../config.h.in config.h)
```

Comment:
Good catch, `HAVE_UDAT_STANDALONE_SHORTER_WEEKDAYS` also needs to be 1
The issue is that the CMakeLists.txt in the upper directory has `include(configure.cmake)`, which uses `CMAKE_CURRENT_SOURCE_DIR` and `CMAKE_CURRENT_BINARY_DIR` which cannot be modified and I didnt want to touch the parent CMakefiles.txt, so, I had replaced with `configure_file`.  I should reuse configure_cmake without duplicating it.

Would it be okay to change the parent CMakeLists.txt:

from:
```
configure_file(
    ${CMAKE_CURRENT_SOURCE_DIR}/config.h.in
    ${CMAKE_CURRENT_BINARY_DIR}/config.h)
```
to 
```
if(NOT DEFINED CMAKE_SOURCE_DIR_VAR)
   set(CMAKE_SOURCE_DIR_VAR, ${CMAKE_CURRENT_SOURCE_DIR})
endif()

if(NOT DEFINED CMAKE_BINARY_DIR_VAR)
   setCMAKE_BINARY_DIR_VAR, ${CMAKE_CURRENT_BINARY_DIR})
endif()

configure_file(
    ${CMAKE_SOURCE_DIR_VAR}/config.h.in
    ${CMAKE_BINARY_DIR_VAR}/config.h)
```
With above change, local-build/build.sh can invoke cmake with
  `-DCMAKE_SOURCE_DIR_VAR="../"` and `-DCMAKE_BINARY_DIR_VAR="../"`

Reply:
Also, can I set `CLR_CMAKE_TARGET_UNIX=1` or do I need to include configureplatform.cmake

Reply:
It seems unnecessarily complicated, let me take a look at the build changes to understand why we need to have the config.h in a different directory.

Reply:
I wonder - wouldn't it be simpler to modify the creation of the package so that we copy everything from the original (src/native/libs/System.Globalization.Native) directory except of the CMakeLists.txt and then copy this new CMakeLists.txt there too (to the same directory)? That way, you would not have problems with the configuration.
Alternatively, instead of having a separate CMakeLists.txt for the local build, you could reuse the original one and just conditionalize it based on a cmake variable (you could use the STATIC_SHIM_COMPILE that you already have)  that we would pass to cmake using the -D option in the new build.sh. That way you wouldn't even need to duplicate the list of source files.

Reply:
>you could reuse the original one and just conditionalize it based on a cmake variable
Sounds good, I will try that, sounds like Jan is onboard as well.
CLR_CMAKE_TARGET_UNIX might still need to be set.

Reply:
>I wonder - wouldn't it be simpler to modify the creation of the package so that we copy everything 
Jan wanted to avoid copying in an earlier patch

Reply:
I wanted avoid copying at build time on the target machine. I would be fine with doing the renaming when we create the package.

> > you could reuse the original one and just conditionalize it based on a cmake variable

> Sounds good, I will try that, sounds like Jan is onboard as well.

Yes, I like this option the most. It is the most sustainable option long term, especially if this gets expanded to other native libraries.

Reply:
>Yes, I like this option the most. It is the most sustainable option long term, especially if this gets expanded to other native libraries.

Thanks, incorporated change to use only `src/native/libs/System.Globalization.Native/CMakeLists.txt`
Conditionalized above CMakeLists.txt for local build, moved the build.sh into parent as local_build.sh and removed local-build directory.

```
ubuntu@ip-10-0-0-223:~/hello$ cat ./obj/Debug/net7.0/linux-arm64/libs/System.Globalization.Native/build/config.h
#pragma once

#define HAVE_UDAT_STANDALONE_SHORTER_WEEKDAYS 1
#define HAVE_SET_MAX_VARIABLE 1
```